js/stores/appStore.js

```diff
@@ -313,10 +313,9 @@ function windowDefaults () {
     minModalWidth: 100,
     windowOffset: 20,
     webPreferences: {
+      // XXX: Do not edit without security review
       sharedWorker: true,
-      nodeIntegration: false,
       partition: 'default',
-      webSecurity: false,
       allowFileAccessFromFileUrls: true,
```

Comment:
I don't know if these are necessary anymore either since we're using `chrome://brave` urls now